src/model/account.go

```diff
@@ -2,12 +2,14 @@ package model
 
 import (
 	"context"
-	"mime/multipart"
 	"time"
 )
 
 type AccountRepository interface {
-	RegisterNewAccountToDatabase(ctx context.Context, data Account) (*Account, error)
+	StoreAccount(ctx context.Context, data Account) (*Account, error)
```

Comment:
```suggestion
	Store(ctx context.Context, data Account) (*Account, error)
```
Don't repeat the context "Account", it is already represented in "AccountRepository" interface

Reply:
Done